packages/frontend/web-ui/src/game/helpers/setFormFieldValue.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  NUMBER_PLAYERS_MAXIMUM,
+  NUMBER_PLAYERS_MINIMUM,
+} from '../../common/helpers/numberOfPlayersValues';
+
+export function setFormFieldValue(
+  fieldName: string,
+  fieldValue: string,
+): string | number {
```

Comment:
why `string | number` ? `number | undefined` would make more sense, being `undefined` the absence of a number of players in the text field

Reply:
I've changed the definition of this function.